org-roam-macs.el

```diff
@@ -58,11 +58,10 @@ to look.
                         (error-message-string err)
                         ,templates))))
 
-(defmacro org-roam-message (format-string &rest args)
-  "Message MSG with ARGS when `org-roam-verbose' is true."
-  (declare (indent 0) (debug t))
```

Comment:
Any reason to delete the `declare` form?  Was it because it was the default?

Reply:
Yeah, I think it's the default, so we can avoid it indeed.

Reply:
`org-roam-message` is a function now. No need for the declare form.